honeybee_radiance/lightsource/sky/hemisphere.py

```diff
@@ -0,0 +1,117 @@
+"""Radiance sky hemisphere."""
+import honeybee_radiance.typing as typing
+import ladybug.futil as futil
+import os
+
+
+class Hemisphere(object):
+    """Radiance sky hemisphere."""
+
+    def __init__(self):
+        """Create sky hemisphere.
+
+        Sky hemisphere is an upward facing glow source. Here is the default sky
```

Comment:
If it's a sky hemisphere, doesn't the glow source point downward to illuminate the ground?

Reply:
You are right and this is confusing. I double checked to ensure I'm not making a mistake here. The source is upward facing in the definition.

sky_glow source sky
0
0
4 __0 0 1__ 180

But I'm not sure how this works in `skyfunc`. Will update the docstring to ensure it doesn't confuse other users.